src/main/windows/mainWindow.ts

```diff
@@ -254,8 +254,8 @@ export class MainWindow extends EventEmitter {
                 throw new Error('Provided bounds info file does not validate, using defaults instead.');
             }
             const matchingScreen = screen.getDisplayMatching(savedWindowState);
-            log.debug('matching screen for main window', matchingScreen);
-            if (!(matchingScreen && (isInsideRectangle(matchingScreen.bounds, savedWindowState) || savedWindowState.maximized))) {
```

Comment:
I wonder what happens if you disconnect all screens. I don't think anyone will ever face this, but someone with a desktop pc could be switching screens and suddenly mattermost crashes. 0/5 if we want to even test that.

Reply:
I think it's safe to say we don't support running Mattermost Desktop without any monitors connected :p